openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -183,7 +186,7 @@ def test_queries(self):
         course_home_url(self.course)
 
         # Fetch the view and verify the query counts
-        with self.assertNumQueries(84, table_blacklist=QUERY_COUNT_TABLE_BLACKLIST):
```

Comment:
@cpennington I believe the query count dropped because it's an audit only course and with the new logic queries are saved if there is no verified mode